frontend/app/services/personal-information-service.server.ts

```diff
@@ -68,37 +68,38 @@ const personalInformationApiResponseSchema = z.object({
 
 type PersonalInformationApiResponse = z.infer<typeof personalInformationApiResponseSchema>;
 
-const personalInfoSchema = z.object({
+const personalInfoDtoSchema = z.object({
   clientInfoList: z
     .object({
       id: z.string().optional(),
       category: z.string().optional(),
     })
     .array()
     .optional(),
-  personsNameList: z
+  lastName: z.string().optional(),
+  firstName: z.string().optional(),
+  homeAddress: z
     .object({
-      lastName: z.string().optional(),
-      firstName: z.string().optional(),
+      categoryCode: z.string(),
```

Comment:
I feel like we don't need `categoryCode` because after mapping, the value would be "Home", but this field is already in an object named `homeAddress`.

(same comment for `mailingAddress.categoryCode`.

Reply:
Fixed: 29206a2f127b07482bac775716bc539e0a828f36